doc/specs/#1012 - Show dependencies.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Florencia Zanollo fzanollo/t-fzanollo@microsoft.com
+created on: 2021-05-28
+last updated: 2021-05-28
+issue id: 1012
+---
+
+# Show dependencies
+
+For [#1012](https://github.com/microsoft/winget-cli/issues/1012)
+
+## Abstract
+Several packages require other packages as dependencies. The Windows Package Manager should be able to support declared dependencies and inform the user about any required dependencies.
+
+## Solution Design
+The Windows Package Manager should be able to show package dependency information for each of the four different types of dependencies declared in the [v1.0 manifest schemas](https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/).
+
+* Windows Features
+* Windows Libraries
+* Package Dependencies (same source)
+* External Dependencies
+
+This information will be shown in the commands: show, install and import. Adding to the information already showed. 

```

Comment:
I think that there is at least potential to need work in the following commands:
- install
- show
- upgrade
- uninstall
- validate
- import

In addition to enumerating which commands will begin to acknowledge dependencies, please have a section for each command and detail what will be output.  As an example:

### install
The install command will enumerate the dependencies of the package version being installed as follows:
```
> winget install notepad++
Found Notepad++ [Notepad++.Notepad++]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
This package requires the following dependencies:
- Windows Feature: Hyper-V
- Package: Microsot.WindowsTerminal
Downloading https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v7.9.5/npp.7.9.5.Installer.x64.exe
```

Reply:
Although uninstall might be a long term goal and could probably be skipped for now as we don't even have the information to inform the user.